psi4/src/psi4/libmints/Testing/Temporary/LastTest.log

```diff
@@ -0,0 +1,3 @@
+Start testing: Mar 01 13:09 EST
+----------------------------------------------------------
+End testing: Mar 01 13:09 EST
```

Comment:
this file and the previous added by mistake, right?

Reply:
Yup, they shouldn't have been added.  My bad!